deploy/ndm-cluster-exporter.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: ndm-cluster-exporter
```

Comment:
can the metadata be matched with the same information that is actually on the stable helm charts, like including an appname, labels etc for all the deployment and daemonsets. 

for exporter can the default prometheus labels be added, so that prometheus can automatically scrape for metrics if exporters are installed.

Reply:
@kmova  Done,